services/issue_triage.py

```
@@ -0,0 +1,143 @@
+    owner = os.environ.get("ISSUE_TRIAGE_OWNER", "strikersam")
+    repo = os.environ.get("ISSUE_TRIAGE_REPO", "autonomous-ai-agency")
+    max_issues = int(os.environ.get("ISSUE_TRIAGE_MAX_ISSUES", "10"))
+
+    try:
+        from agent.github_tools import GitHubTools
+        # Autonomous loop, not a per-user request — reads the shared token
+        # directly, matching agent/agency.py and agent/workflow.py.
+        token = os.environ.get("GH_TOKEN") or os.environ.get("GH_PAT") or os.environ.get("GITHUB_TOKEN")
+        client = GitHubTools(token=token)
+        issues = await client.list_issues(owner, repo, state="open", per_page=max_issues)
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)